### DISCUSSION TITLE
自定义Source和Sink

### DISCUSSION THREAD

**Discussion by 71aoo** [tabby-sec/tabby#52] [Q&A] [answered]
大佬，可以自定义Source和sink吗？

**Comment by wh1t3p1g** [comment, marked as answer]
source 推荐在查询的时候制定，常见的web端点可以用`IS_ENDPOINT`
sink 自定义看 rules 规则下面的 sink.json 文件，自行添加即可

**Comment by wh1t3p1g** [comment]
Tabby v2.0 版本已支持自定义source和sink的规则，参见 https://www.yuque.com/wh1t3p1g/tp0c1t/islc2yss3wuvwpg2
